### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by pawelnowicki87**
- [DEMO LINK](https://pawelnowicki87.github.io/layout_calendar/)
- [TEST REPORT LINK](https://pawelnowicki87.github.io/layout_calendar/report/html_report/)

❗️ Copy this `Checklist` to the `Pull Request` description after links, and put `- [x]` before each point after you checked it.

- [x] Changing 'month-lengh' and 'start-day' modifier in the code element
reflects in changing calendar layout
- [x] Each day has no modifiers, only class (eg. calendar__day)
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/styles/blocks/cell.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  border: 1px solid black;
+  background-color: $background-color;
+  position: relative;
+  transition: background-color $animation-duration, transform $animation-duration;
+
+  @for $day from 1 through 31 {
+    &:nth-child(#{$day})::before {
+      content: '#{$day}';
+    }
+
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/cell.scss]:
```suggestion
```

File: src/styles/blocks/cell.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
```
Comment (anastasiiavorobiova) [inline src/styles/blocks/cell.scss]:
```suggestion
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Make sure to deploy your page

**Comment by pawelnowicki87** [comment]
I cant start 'npm run deploy'

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! calendar@1.0.0 deploy: `mate-scripts deploy`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the calendar@1.0.0 deploy script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Paweł\AppData\Roaming\npm-cache\_logs\2023-12-10T20_56_04_166Z-debug.log

File: src/index.html [hunk 3]
```diff
@@ -1,12 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+    name="viewport"
+    content="width=device-width, initial-scale=1.0"
+    >
```
Comment (volodymyr-soltys97) [inline src/index.html:9]:
```suggestion
    <meta
      name="viewport"
      content="width=device-width, initial-scale=1.0"
    >
```

File: src/styles/blocks/cell.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  }
+
+  &:hover {
+    cursor: pointer;
+    background-color: $hovered-color;
+    transform: translateY(-20px);
+  }
+}
+
+
```
Comment (volodymyr-soltys97) [inline src/styles/blocks/cell.scss:26]:
```suggestion
}

```

File: src/styles/utils/_modifiers.scss [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+//   .calendar--start-#{$day} {
+//     margin-left: $cell-size * index($day, mon, tue, wed, thu, fri, sat, sun) + $padding;
+//   }
+// }
+
+// @for $length from 28 through 31 {
+//   .calendar--month-#{$length} .calendar__day:nth-child(#{$length + 1}) ~ .calendar__day {
+//     display: none;
+//   }
+// }
```
Comment (volodymyr-soltys97) [inline src/styles/utils/_modifiers.scss:11]: Remove comments
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Demo link, if you have a problem with deploy feel free in the fe_chat
image

**Pull request closed without merging**